src/CitizenProject/CitizenProjectManager.php

```diff
@@ -85,6 +85,11 @@ public function getCitizenProjectCreator(CitizenProject $citizenProject): ?Adher
         return $this->getAdherentRepository()->findOneByUuid($citizenProject->getCreatedBy());
     }
 
+    public function getCitizenProjectCoordinators(CitizenProject $citizenProject): AdherentCollection
```

Comment:
Wow so much methods in this class whereas the repository already has them :/. What's the point?

Reply:
Yes this PR is in WIP. Thx ;)